concent-builder/install-secrets.yml

```diff
@@ -0,0 +1,50 @@
+- hosts:
+    - concent-builder
+  vars_files:
```

Comment:
Moving secrets from `install-repositories.yml` to `install-secrets.yml` should be in a separate commit.

Also, you should update README so that it says to run this playbook.

Reply:
Done.